src/DotNet.Status.Web/TimelineIssueTriage.cs

```diff
@@ -114,7 +114,10 @@ public async Task ProcessIssueEvent(IssuesHookData issuePayload)
                     update.AddLabel(_markingLabelName);
                     foreach(var label in _issueLabels)
                     {
-                        update.AddLabel(label);
+                        if (issue.Labels.All(l => l.Name != label))
+                        {
+                            update.AddLabel(label);
+                        }
                     }
```

Comment:
```suggestion
                    foreach (var label in _issueLabels.Except(issue.Labels))
                    {
                        update.AddLabel(label);
                    }
```
?

Reply:
Ah, merged already. Never mind then!

Reply:
Cool but ... too late :-) I will fix it in next iteration, if I remember